packages/fiori/src/ShellBar.js

```diff
@@ -16,8 +16,9 @@ import "@ui5/webcomponents-icons/dist/icons/bell.js";
 import "@ui5/webcomponents-icons/dist/icons/overflow.js";
 import "@ui5/webcomponents-icons/dist/icons/grid.js";
 
-// Template
+// Templates
 import ShellBarTemplate from "./generated/templates/ShellBarTemplate.lit.js";
+import ShellBarPopoverFragment from "./generated/templates/ShellBarPopoverFragmentTemplate.lit.js";
```

Comment:
Not strictly against, but ShellBarStaticAreaTemplate.lit.js seems easier to understand